modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -384,6 +384,26 @@ public Method getFunctionalMethod() {
     }
 
     public static class Cast {
+        public static Cast standard(Type from, Type to, boolean explicit) {
```

Comment:
If we're going to have these helper methods, please at least give a one line javadoc explaining each one.

Reply:
Done.